### DISCUSSION TITLE
Next.js 14 + Supabase SSR with Tailwind CSS

### DISCUSSION THREAD

**Discussion by hyderbeck** [realworld-apps/realworld#1536] [WIP Implementations]
Hey everyone! The app's already [deployed](https://conduit-gamma.vercel.app). It's fully-functioning, built with Next.js' latest capabilities, but not all the required features are implemented yet, so I'd be happy to collaborate if you can bear with the redesign. I started with rewriting the theme in Tailwind, but somehow ended up giving a completely different (and unfortunate) look to it.

Users may create an account, add & edit articles, browse feed, follow each other, among other things. I've been keeping a [project](https://github.com/users/hyderbeck/projects/10) mirroring my commit history, and all the todos are there. Criticism is most welcome.

**Comment by geromegrignon** [comment]
Hi @hyderbeck ,

Thanks for your interest in this project.
I hope you had a great experience trying out another theme: however the theme is part of the specs so rewriting them does not make the implementation compliant with the expectations to list it on our official listing.
